app/controllers/admin/static_pages/static_pages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Admin::StaticPages::StaticPagesController < ApplicationController
```

Comment:
I think this is module-nested correctly? I want the route to read /admin/static_pages/learn_more when they are editing on the CMS page.